src/backend/marsha/markdown/permissions.py

```diff
@@ -32,26 +33,25 @@ def _is_playlist_admin(user_id, markdown_document_id):
     ).exists()
 
 
-class IsTokenResourceRouteObjectRelatedResource(permissions.BasePermission):
+class IsTokenResourceRouteObjectRelatedMarkdownDocument(permissions.BasePermission):
     """
-    Base permission class for JWT Tokens related to a resource object linked to a resource.
+    Base permission class for JWT Tokens related to a resource object linked to a
+    Markdown document.
 
-    These permissions grant access to users authenticated with a resource JWT token built from a
-    resource.
+    These permissions grants access to users authenticated with a JWT token built from a
+    resource ie related to a TokenUser as defined in `rest_framework_simplejwt`.
     """
 
-    linked_resource_attribute = ""
-
     def has_permission(self, request, view):
         """
-        Allow the request if the JWT resource matches the resource
-        related to the object in the url.
+        Allow the request if the JWT resource matches the Markdown document related to the object
+        in the url.
 
         Parameters
         ----------
-        request : Type[rest_framework.request.Request]
+        request : Type[django.http.request.HttpRequest]
             The request that holds the authenticated user
-        view : Type[rest_framework.viewsets or rest_framework.views]
+        view : Type[restframework.viewsets or restframework.views]
```

Comment:
why are you changing `rest_framework` by `restframework` ? the module name is `rest_framework`